Makefile

```diff
@@ -3,6 +3,15 @@ $(shell cp -n \.env.default \.env)
 $(shell cp -n \.\/src\/docker\/docker-compose\.override\.yml\.default \.\/src\/docker\/docker-compose\.override\.yml)
 include .env
 
+# Setup PHP Variables based on package version
+
+PHP_IMAGE=php:$(shell printf '%s' "$(PHP_VERSION)" | sed -e 's/\.//')-fpm
+PHP_CONFIG=/etc/php$(PHP_VERSION)
```

Comment:
I think PHP_CONFIG  var is useless
we can use /etc/php${PHP_VERSION} in docker-compose.override.yml.default